src/Controller/ApiController.php

```diff
@@ -386,8 +386,16 @@ public function projectStatsAction(Request $request)
         $data = $client->search([
             'body' => [
                 'query' => [
-                    'match' => [
-                        'project.id' => $projectId,
+                    'constant_score' => [
+                        'filter' => [
+                            'bool' => [
+                                'must' => [
+                                    'term' => [
+                                        'project.id' => $projectId,
+                                    ],
+                                ],
+                            ],
+                        ],
```

Comment:
Aku vyhodu ma to `constant_score`?

Reply:
No to mam nekde z docu - popsane je to tady https://www.elastic.co/guide/en/elasticsearch/reference/current/compound-queries.html je to proto, ze jsem tam upravil ten match (fuzzy matching) na term (exact) a timpadem maji mit vsechny dokumenty stejny score a neni potreba ho pocitat